src/_includes/core-nodes/tls-use-case.md

```diff
@@ -0,0 +1,32 @@
+## What is TLS config node in Node-RED?
+
+In Node-RED, the TLS Config node is used to configure Transport Layer Security (TLS) settings for secure communication over networks. TLS ensures that data transmitted between applications, servers, devices is encrypted and secure. However, this node is not directly available in the Node-RED palette; it is accessible within Node-RED core nodes and custom nodes used for communicating with other services over the internet.
```

Comment:
I've added this to inform users that they don't need to drag it from the Node-RED palette; it won't be available there. Instead, it will be available in the network node configuration settings.